project/Dependencies.scala

```diff
@@ -92,19 +92,19 @@ object Dependencies {
     // End-to-end test specs
     specs2 % Test,
     // Writing XML. import com.sun.xml.txw2
-    "org.glassfish.jaxb" % "txw2" % "4.0.2",
+    "org.glassfish.jaxb" % "txw2" % "4.0.5",
     // Makes txw2 write self-closing tags in xml (which we want). Not imported.
     "org.codehaus.woodstox" % "wstx-asl" % "4.0.6",
     // Json Web Tokens (used for OIDC Auth). import pdi.jwt
-    "com.github.jwt-scala" %% "jwt-play-json" % "9.2.0",
+    "com.github.jwt-scala" %% "jwt-play-json" % "10.0.0",
```

Comment:
https://github.com/jwt-scala/jwt-scala/releases says no breaking changes, just the play 3 upgrade